sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -58,6 +58,14 @@ SELECT substring('Spark SQL' from 5);
 SELECT substring('Spark SQL' from -3);
 SELECT substring('Spark SQL' from 5 for 1);
 
+-- mask function
+select mask_hash('TestString-123'),
```

Comment:
also add a case where the input is NULL?

Reply:
Done